review.txt

```diff
@@ -0,0 +1,5 @@
+Please find below review comments.
+ code restructure 
+     - divide code in folder like map. footer, table ......
+     - do not write style in html file
```

Comment:
Also do not hard code API url in APP.js, use config/const  file for that.
as it may change